src/reducers/RecipeReducer.js

```diff
@@ -5,6 +5,9 @@ const initialState = {
   recipes: [],
   recipeDetails: {},
   recipeRecs: [],
+  likedRecipes: [],
+  recipeToLike: {},
```

Comment:
It doesn't make sense to have recipetoLike or unLike. The back end just updates the recipe by id to be liked or unliked. I believe the use case is in the details page, so probably have the respective actions with fetch and receive the recipeDetails of the recipe in question. @mavalos1 please provide your thoughts

Reply:
We should keep the action types to differentiate the action clearly, but i eliminated those two properties and pointed them to the recipeDetails property instead. Please let me know if that is cool with you.